languages/go/exercises/concept/constants/.docs/after.md

```diff
@@ -0,0 +1,47 @@
+Constants in Go are simple, [unchanging values][const] created with the `const` keyword. Constants may be given an explicit type:
```

Comment:
We've recently [updated the spec](https://github.com/exercism/v3/issues/2293) (although we haven't updated the docs yet) which results in `after.md` documents being replaced with concept documents. Concretely, this file should be moved to `languages/go/concepts/constants/about.md`.